create_graphics.py

```diff
@@ -445,12 +445,15 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
 
     fig, ax = plt.subplots(1, 1, figsize=(inches, inches))
 
+    plot_airports = spec.get('plot_airports', True)
```

Comment:
This is a pretty short call to spec. You could put it directly in the Map object creation on line 456 to reduce the number of variables created in this method.